### ISSUE TITLE
cannot load 

### ISSUE THREAD

**Issue by JenJenJenkins** [Questie/Questie#3137]
## Bug description
it is a lua message. Even if I activate only Questie, it ist alway saying:

Date: 2021-08-21 12:41:36
ID: 1
Error occured in: Global
Count: 1
Message: ...nterface\AddOns\Questie\Modules\Tooltips\Tooltip.lua line 51:
   attempt to index local 'npc' (a nil value)
Debug:
   [string "@RCLootCouncil_Classic\RCLootCouncil\Core\ErrorHandler.lua"]:146:
      ...tCouncil_Classic\RCLootCouncil\Core\ErrorHandler.lua:144
   [string "=[C]"]: ?
   [string "@Questie\Modules\Tooltips\Tooltip.lua"]:51: RegisterQuestStartTooltip()
   [string "@Questie\Modules\Quest\QuestieQuest.lua"]:1327: DrawAvailableQuest()
   [string "@Questie\Modules\Quest\QuestieQuest.lua"]:1461:
      ...erface\AddOns\Questie\Modules\Quest\QuestieQuest.lua:1433
   [string "@..\SharedXML\C_TimerAugment.lua"]:16:
      ..\SharedXML\C_TimerAugment.lua:14
Locals:
private = <table> {
 ThrowError = <function> defined @Interface\AddOns\RCLootCouncil_Classic\RCLootCouncil\Core\ErrorHandler.lua:46
 DoesErrorExist = <function> defined @Interface\AddOns\RCLootCouncil_Classic\RCLootCouncil\Core\ErrorHandler.lua:87
 ClearOldErrors = <function> defined @Interface\AddOns\RCLootCouncil_Classic\RCLootCouncil\Core\ErrorHandler.lua:107
 SanitizeLine = <function> defined @Interface\AddOns\RCLootCouncil_Classic\RCLootCouncil\Core\ErrorHandler.lua:83
 MAX_STACK_DEPTH = 10
 log = <table> {
 }
 ErrorHandler = <function> defined @Interface\AddOns\RCLootCouncil_Classic\RCLootCouncil\Core\ErrorHandler.lua:116
 IncrementErrorCount = <function> defined @Interface\AddOns\RCLootCouncil_Classic\RCLootCouncil\Core\ErrorHandler.lua:77
 NewError = <function> defined @Interface\AddOns\RCLootCouncil_Classic\RCLootCouncil\Core\ErrorHandler.lua:50
 IsRCLootCouncilError = <function> defined @Interface\AddOns\RCLootCouncil_Classic\RCLootCouncil\Core\ErrorHandler.lua:94
}
orig_errorhandler = <function> defined @Interface\AddOns\Decursive\Dcr_DIAG.lua:609

AddOns:
  Swatter, v1.13.6652 (SwimmingSeadragon)
  Altoholic, vr191
  AtlasLootClassic, vv2.3.1-bcc
  AtlasLootClassicData, vv2.3.1-bcc
  AtlasLootClassicDungeonsAndRaids, vv2.3.1-bcc
  Attune, v238
  AucAdvanced, v1.13.6717 (SwimmingSeadragon)
  AucFilterBasic, v1.13.6680 (SwimmingSeadragon)
  AucStatHistogram, v1.13.6681 (SwimmingSeadragon)
  AucStatiLevel, v1.13.6705 (SwimmingSeadragon)
  AucStatPurchased, v1.13.6672 (SwimmingSeadragon)
  AucStatSimple, v1.13.6634 (SwimmingSeadragon)
  AucStatStdDev, v1.13.6635 (SwimmingSeadragon)
  AucUtilFixAH, v1.13.6608 (SwimmingSeadragon)
  BagBrother, v
  Bagnon, v9.1.0
  Bartender4, v4.10.10
  BeanCounter, v1.13.6682 (SwimmingSeadragon)
  BuffomatClassicTBC, v2021.7.5-6abbc1de
  DataStore, vr66
  DataStoreAgenda, vr30
  DataStoreAuctions, vr51
  DataStoreCharacters, vr44
  DataStoreContainers, vr60
  DataStoreCrafts, vr83
  DataStoreInventory, vr54
  DataStoreMails, vr53
  DataStoreQuests, vr47
  DataStoreReputations, vr44
  DataStoreSpells, vr33
  DataStoreTalents, vr62
  DBMCore, v2.5.10
  DBMStatusBarTimers, v
  Decursive, v2.7.8.3
  Details, v
  DetailsStreamer, v
  DetailsTinyThreat, v
  Enchantrix, v1.13.6708 (SwimmingSeadragon)
  EnchantrixBarker, v1.13.6697 (SwimmingSeadragon)
  FishingBuddy, v0.7.5.5
  GatherMate2, v1.46.1-classic
  GTFO, v4.63
  Guildbook, v4.96521
  Informant, v1.13.6684 (SwimmingSeadragon)
  LeatrixMaps, v2.5.50
  LeatrixPlus, v2.5.53
  LeatrixSounds, v2.5.48
  LFGBulletinBoard, v2.59
  Outfitter, v2.0.4
  Pawn, v2.5.24
  Prio3, v20210521
  QoLClassicThreatTBC, v10
  QoLCoreTBC, v5
  Questie, v6.5.1
  RCLootCouncilClassic, v0.13.0
  Reputable, v1.26-bcc
  SilverDragon, vv20501.0
  SilverDragonBurningCrusade, vv20501.0
  SilverDragonClassic, vv20501.0
  SlideBar, v1.13.6668 (SwimmingSeadragon)
  Stubby, v1.13.6614 (SwimmingSeadragon)
  ThreatClassic2, v2.38
  TidyPlatesThreatPlates, v10.2.11
  TitanClassic, v1.3.4.20501
  TitanClassicAmmo, v1.3.4.20501
  TitanClassicBag, v1.3.4.20501
  TitanClassicClock, v1.3.4.20501
  TitanClassicGold, v1.3.4.20501
  TitanClassicLocation, v1.3.4.20501
  TitanClassicLootType, v1.3.4.20501
  TitanClassicPerformance, v1.3.4.20501
  TitanClassicRegen, v1.3.4.20501
  TitanClassicRepair, v1.3.4.20501
  TitanClassicVolume, v1.3.4.20501
  TitanClassicXP, v1.3.4.20501
  VendorPrice, v1.4.0
  VuhDo, v3.124-tbcc
  VuhDoOptions, v3.124-tbcc
  ZPerl, v6.2.4
  ZPerlArcaneBar, v
  ZPerlParty, v
  ZPerlPartyPet, v
  ZPerlPlayer, v
  ZPerlPlayerBuffs, v
  ZPerlPlayerPet, v
  ZPerlRaidAdmin, v
  ZPerlRaidFrames, v
  ZPerlRaidHelper, v
  ZPerlRaidMonitor, v
  ZPerlRaidPets, v
  ZPerlTarget, v
  ZPerlTargetTarget, v
  BlizRuntimeLib_deDE v2.5.1.20501 <none>
  (ck=a7c)

## Screenshots
![bug questie](https://user-images.githubusercontent.com/89300355/130321261-f7fe1243-5077-46b3-a2f8-7000533d206e.PNG)

## Questie version
I made sure I have the lastest version, the Questei.toc file says it's ## Version: 6.5.1and it ist still not running.

**Comment by JenJenJenkins** [comment]
not sure, if thats the right way to report, sry

**Comment by Laumesis** [comment]
Please try following first to see if it fixes the issue:
Delete `Questie` and `QuestieDev-master` (if you have one) in your `AddOns` folder, then reinstall Questie.
Delete first, don't replace/overwrite.
Then restart the game. 
Then open Questie's Options (`/questie` command) and go to Advanced tab. There click `Recompile Database`.

**Comment by JenJenJenkins** [comment]
Hi, thank you for the fast reply. I tried it but it changed only the lua line in the message ;)
--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
Date: 2021-08-22 19:25:10ID: 1Error occured in: GlobalCount: 1Message: ...lizzard_MapCanvas\MapCanvas_ScrollContainerMixin.lua line 428:   attempt to index local 'layers' (a nil value)Debug:   [string ***@***.***_Classic\RCLootCouncil\Core\ErrorHandler.lua"]:146:      ...tCouncil_Classic\RCLootCouncil\Core\ErrorHandler.lua:144   [string "=[C]"]: ?   [string ***@***.***_MapCanvas\MapCanvas_ScrollContainerMixin.lua"]:428: CreateZoomLevels()   [string ***@***.***_MapCanvas\MapCanvas_ScrollContainerMixin.lua"]:415: OnCanvasSizeChanged()   [string ***@***.***_MapCanvas\MapCanvas_ScrollContainerMixin.lua"]:407: SetMapID()   [string ***@***.***_MapCanvas\Blizzard_MapCanvas.lua"]:35: SetMapID()   [string ***@***.***\TitanClassicLocation.lua"]:75: TitanPanelLocationButton_OnShow()   [string "*:OnShow"]:1:      [string "*:OnShow"]:1   [string "=[C]"]: Show()   [string ***@***.***\TitanPanelClassic.lua"]:1475: TitanPanel_InitPanelButtons()   [string ***@***.***\TitanClassicVariables.lua"]:1045: TitanVariables_UseSettings()   [string ***@***.***\TitanPanelClassic.lua"]:400:      TitanClassic\TitanPanelClassic.lua:327   [string "=[C]"]: pcall()   [string ***@***.***\TitanPanelClassic.lua"]:467: ?()   [string ***@***.***\TitanPanelClassic.lua"]:309:      TitanClassic\TitanPanelClassic.lua:308Locals:private = <table> { ThrowError = <function> defined @Interface\AddOns\RCLootCouncil_Classic\RCLootCouncil\Core\ErrorHandler.lua:46 DoesErrorExist = <function> defined @Interface\AddOns\RCLootCouncil_Classic\RCLootCouncil\Core\ErrorHandler.lua:87 ClearOldErrors = <function> defined @Interface\AddOns\RCLootCouncil_Classic\RCLootCouncil\Core\ErrorHandler.lua:107 SanitizeLine = <function> defined @Interface\AddOns\RCLootCouncil_Classic\RCLootCouncil\Core\ErrorHandler.lua:83 MAX_STACK_DEPTH = 10 log = <table> { } ErrorHandler = <function> defined @Interface\AddOns\RCLootCouncil_Classic\RCLootCouncil\Core\ErrorHandler.lua:116 IncrementErrorCount = <function> defined @Interface\AddOns\RCLootCouncil_Classic\RCLootCouncil\Core\ErrorHandler.lua:77 NewError = <function> defined @Interface\AddOns\RCLootCouncil_Classic\RCLootCouncil\Core\ErrorHandler.lua:50 IsRCLootCouncilError = <function> defined @Interface\AddOns\RCLootCouncil_Classic\RCLootCouncil\Core\ErrorHandler.lua:94}orig_errorhandler = <function> defined @Interface\AddOns\Decursive\Dcr_DIAG.lua:609
AddOns:  Swatter, v1.13.6652 (SwimmingSeadragon)  Altoholic, vr191  AtlasLootClassic, vv2.3.1-bcc  AtlasLootClassicData, vv2.3.1-bcc  AtlasLootClassicDungeonsAndRaids, vv2.3.1-bcc  Attune, v239  AucAdvanced, v1.13.6717 (SwimmingSeadragon)  AucFilterBasic, v1.13.6680 (SwimmingSeadragon)  AucStatHistogram, v1.13.6681 (SwimmingSeadragon)  AucStatiLevel, v1.13.6705 (SwimmingSeadragon)  AucStatPurchased, v1.13.6672 (SwimmingSeadragon)  AucStatSimple, v1.13.6634 (SwimmingSeadragon)  AucStatStdDev, v1.13.6635 (SwimmingSeadragon)  AucUtilFixAH, v1.13.6608 (SwimmingSeadragon)  BagBrother, v  Bagnon, v9.1.0  Bartender4, v4.10.10  BeanCounter, v1.13.6682 (SwimmingSeadragon)  BuffomatClassicTBC, v2021.7.5-6abbc1de  DataStore, vr66  DataStoreAgenda, vr30  DataStoreAuctions, vr51  DataStoreCharacters, vr44  DataStoreContainers, vr60  DataStoreCrafts, vr83  DataStoreInventory, vr54  DataStoreMails, vr53  DataStoreQuests, vr47  DataStoreReputations, vr44  DataStoreSpells, vr33  DataStoreTalents, vr62  DBMCore, v2.5.10  DBMStatusBarTimers, v  Decursive, v2.7.8.3  Details, v  DetailsStreamer, v  DetailsTinyThreat, v  Enchantrix, v1.13.6708 (SwimmingSeadragon)  EnchantrixBarker, v1.13.6697 (SwimmingSeadragon)  FishingBuddy, v0.7.5.5  GatherMate2, v1.46.1-classic  GTFO, v4.63  Guildbook, v4.96522  Informant, v1.13.6684 (SwimmingSeadragon)  LeatrixMaps, v2.5.50  LeatrixPlus, v2.5.53  LeatrixSounds, v2.5.48  LFGBulletinBoard, v2.59  Outfitter, v2.0.4  Pawn, v2.5.25  Prio3, v20210521  QoLClassicThreatTBC, v10  QoLCoreTBC, v5  Questie, v6.5.1  RCLootCouncilClassic, v0.13.0  Reputable, v1.26-bcc  SilverDragon, vv20501.0  SilverDragonBurningCrusade, vv20501.0  SilverDragonClassic, vv20501.0  SlideBar, v1.13.6668 (SwimmingSeadragon)  Stubby, v1.13.6614 (SwimmingSeadragon)  ThreatClassic2, v2.38  TidyPlatesThreatPlates, v10.2.11  TitanClassic, v1.3.4.20501  TitanClassicAmmo, v1.3.4.20501  TitanClassicBag, v1.3.4.20501  TitanClassicClock, v1.3.4.20501  TitanClassicGold, v1.3.4.20501  TitanClassicLocation, v1.3.4.20501  TitanClassicLootType, v1.3.4.20501  TitanClassicPerformance, v1.3.4.20501  TitanClassicRegen, v1.3.4.20501  TitanClassicRepair, v1.3.4.20501  TitanClassicVolume, v1.3.4.20501  TitanClassicXP, v1.3.4.20501  VendorPrice, v1.4.0  VuhDo, v3.124-tbcc  VuhDoOptions, v3.124-tbcc  ZPerl, v6.2.4  ZPerlArcaneBar, v  ZPerlCustomHighlight, v  ZPerlParty, v  ZPerlPartyPet, v  ZPerlPlayer, v  ZPerlPlayerBuffs, v  ZPerlPlayerPet, v  ZPerlRaidAdmin, v  ZPerlRaidFrames, v  ZPerlRaidHelper, v  ZPerlRaidMonitor, v  ZPerlRaidPets, v  ZPerlTarget, v  ZPerlTargetTarget, v  BlizRuntimeLib_deDE v2.5.1.20501 <none>  (ck=a96)
--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------

I learned it is pretty hard, to quest without your addon :(



    Am Sonntag, 22. August 2021, 12:24:53 MESZ hat Laume ***@***.***> Folgendes geschrieben:  
Please try following first to see if it fixes the issue:

Delete Questie and QuestieDev-master (if you have one) in your AddOns folder, then reinstall Questie.
Delete first, don't replace/overwrite.
Then restart the game.
Then open Questie's Options (/questie command) and go to Advanced tab. There click Recompile Database.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by drejjmit** [comment]
Are you even on Classic wow? That's a lot of errors from all your addons.

**Comment by JenJenJenkins** [comment]
I deleted the Questie folder in C:\Spiele\World of
Warcraft\_classic_\Interface\Addons and now I searched the addon new on
wowup.
Very interesting is, there is a Version called Questie v6.5.1 (described as
6.5.1) and Questie (described as v6.5.1 - TBC)  available.
I had  Questie v6.5.1, and it worked the last weeks. Now it doesn't work
anymore, but the Questie ist working fine now.
Thank you very much for your help.

Am So., 22. Aug. 2021 um 19:44 Uhr schrieb drejjmit <
***@***.***>:
